FLEDGE.md

```diff
@@ -132,7 +132,7 @@ The browser will only allow the `joinAdInterestGroup()` operation with the permi
 
 The returned `joinPromise` is resolved if the group is successfully joined, and rejected with an error if the join operation fails. The error message and the resolution time must _not_ depend on what interest groups a user is in, or any cross-origin browser state, apart from the results of the .well-known fetch, to avoid leaking any data across sites.
 
-There is a complementary API `navigator.leaveAdInterestGroup(myGroup)` which looks only at `myGroup.name` and `myGroup.owner`. As with join calls, `leaveAdInterestGroup()` also returns a promise. As a special case to support in-ad UIs, invoking `navigator.leaveAdInterestGroup()` from inside an ad that is being targeted at a particular interest group will cause the browser to leave that group, irrespective of permission policies. Note that calling `navigator.leaveAdInterestGroup()` without arguments isn't supported inside a component ad frame.
```

Comment:
Note: This is the sole occurrence of ["component ad frame"](https://github.com/search?q=repo%3AWICG%2Fturtledove+%22component+ad+frame%22&type=code) in the source code of this repository, so it isn't clear to me what is the definition of it. Do you know?

Reply:
Added by another Googler minutes ago...
https://github.com/WICG/turtledove/commit/2a455660a55dd81f22e5a0ef04915cfa49d45fd8

Reply:
Thanks!